notebooks/02_analyses/sentiment_analysis.ipynb

```diff
@@ -0,0 +1,776 @@
+{
```

Comment:
<p>Would it make sense to see a plot of how the sentiment <em>changes</em> over months? I'm thinking this might show some interesting "spikes" in certain months </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/66/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>